### PR TITLE
Add python apidaora

### LINKED ISSUE #1936: Add apidaora
Hi @dutradda,

Would you mind to add (or consent that we add) [apidora](https://github.com/dutradda/apidaora) in `here` ?

This project compare performance based on 3 routes :

+ `GET` `/`, **SHOULD** return a successful status code and an empty string
+ `GET` `/user/:id`, **SHOULD** return a successful status code and the id
+ `POST` `/user`, **SHOULD** return a successful status code and an empty string

Regards,

### PR DESCRIPTION
**Pull request by douglasfarinelli**
Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [x] A `Dockerfile` exists, or can be built ?
+ [X] I've updated `CI` config file (and **commited** `.travis.yml`)
~~~sh
bin/make ci_config
~~~
+ [X] I've updated `.dependabot` config file  (and **commited** `.dependabot/config.yml`)
~~~sh
bin/make deps_config
~~~
+ [x] All tests passes ?
~~~
export FRAMEWORK=<MY_FRAMEWORK>
shards install
shards build --release
bin/make config
bin/neph ${FRAMEWORK}
crystal spec
~~~

-----
edit : Closes #1936

### CODE REVIEW

File: python/apidaora/config.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+framework:
+  github: dutradda/apidaora
+  version: 0.11.4
```
Comment (waghanza) [inline python/apidaora/config.yaml]: please use X.Y format, since `0.11.8` (example) could be installed according to requirements

**Comment by waghanza** [comment]
@douglasfarinelli @dutradda We can **NOT** add `apidaora` yet. We are using `python` `3.7` on containers and `apidaora` require  `python` `3.8`.
https://github.com/dutradda/apidaora/blob/0.11.4/pyproject.toml#L22

**Comment by douglasfarinelli** [comment]
> @douglasfarinelli @dutradda We can **NOT** add `apidaora` yet. We are using `python` `3.7` on containers and `apidaora` require `python` `3.8`.
> https://github.com/dutradda/apidaora/blob/0.11.4/pyproject.toml#L22

No problems! I saw it today haha

How can we proceed @waghanza?

**Comment by dutradda** [comment]
@waghanza I can help you to update the python version

**Comment by waghanza** [comment]
@dutradda It is **NOT** a matter of that, but having a version used in **production** for `python`, btw this feature is in the pipe in https://github.com/the-benchmarker/web-frameworks/pull/1951
however, when results will be **production-ready**, we will use various of platforms to display results (`kubernetes`, `baremetal` ...)
for `kubernetes` we have **NOT** the problem, since we can use any version, but for others we need to stick to system version, for example only `python` **3.6** is available on `centos`
https://www.softwarecollections.org/en/scls/?search=python

why `apidaora` requires `python` **3.8** ? is it requiring some feature **NOT** in `3.6` or `3.7` ?

**Comment by dutradda** [comment]
@waghanza 
apidaora uses in its core the [TypedDict](https://www.python.org/dev/peps/pep-0589/) class, which only available on python3.8

**Comment by dutradda** [comment]
@waghanza 
I will do a backport implementation for python >= 3.5 < 3.8 using the [typing-extensions](https://pypi.org/project/typing-extensions/) module.

I will keep you informed

**Comment by waghanza** [comment]
@dutradda any changes ?

**Comment by waghanza** [comment]
@dutradda @douglasfarinelli `apidaora` requires `rust` to be built ?

**Comment by waghanza** [comment]
I've found a trick https://github.com/ijl/orjson/issues/8#issuecomment-526621349 to install `apidaora` on **alpine**

**Comment by waghanza** [comment]
@dutradda @douglasfarinelli I have enabled `apidaora` `here`

**Pull request merged**